packages/next/src/server/app-render/action-handler.ts

```diff
@@ -299,6 +299,9 @@ export async function handleAction({
     )
   }
 
+  // When running actions the default is no-store, you can still `cache: 'force-cache'`
```

Comment:
can you add a test for the force-cache scenario>

Reply:
Done!